package.json

```diff
@@ -16,6 +16,7 @@
     "@bufbuild/protobuf": "1.10.0",
     "@bufbuild/protoc-gen-es": "1.10.0",
     "@microsoft/ccf-app": "6.0.0",
+    "cbor-x": "1.6.0",
```

Comment:
Can we parse the header ourself or use a library made by a team close to us? While it's great there is such library, I think it's preferable if we can avoid external libraries to avoid situations like where we face a bug but we need to wait for fix long time etc.

Reply:
If it was trivial to do either, I would also prefer that for the reasons you mentioned. Unfortunately I don’t think that is the case here. If I remember correctly from before I went away, @KenGordon was okay with us using a cbor library?

Reply:
Obviously I would rather not but I don't think we want to be redoing this sort of thing ourselves either. It happens that these ought to be pretty simple so parsing a necessary subset would not be hard. However, plans like that often end up being fragile and so much more work in the end. On balance, I think this is ok.